web/components/promtools-output.js

```diff
@@ -0,0 +1,31 @@
+import {css, html, LitElement} from 'https://unpkg.com/lit-element@2.3.1/lit-element.js?module';
+
+class Output extends LitElement {
+    static get properties() {
+        return {
+            output: {type: String},
+        }
+    }
+
+    static get styles() {
+        return css`
+            pre {
+                padding: 1.25rem 1.5rem;
+                background-color: #f5f5f5;
+                border:1px solid #ddd;
+                border-left: 4px solid #e6522c;
+                overflow-x: auto;
+            }
+        `;
+    }
+
+    render() {
+        return html`
+            ${this.output === '' ? html`` : html`
```

Comment:
I tend to replace html`` with `nothing` directive from lit-html https://lit-html.polymer-project.org/guide/writing-templates#rendering-nothing , I can't tell from the docs why it would be an advantage to use this, but they seem to encourage it so I'm sure they have a reason for doing so that's not just syntactical.

Reply:
Right. I think I tried that ones and had some problems getting the correct import. Will try again :+1:

Reply:
Just an empty `''` here would be enough, `nothing` is used in combination with slots, which is not the case here

Reply:
So even '' as opposed to html``? It will still render fine? It doesn't return a TemplateResult then, right? Still renders and even re-renders fine?

Reply:
yep